src/modules/common/components/LastUpdateInfo.js

```diff
@@ -16,18 +16,20 @@ const TimeStamp = styled.p`
 type PropsType = {|
   lastUpdate: Moment,
   t: TFunction,
-  language: string
+  language: string,
+  withText: boolean,
+  className?: string
```

Comment:
Why do you need the className?

Reply:
Good morning @Taggotty

I passed the `className` to allow me tweak and customize the component style if needed using styled components
example:
```js
const CustomLastUpdateInfo = styled(LastUpdateInfo)`
  padding: 0 10px;
  color: ${({ theme }) => (theme.colors.backgroundColor)};

  &&& {
    margin: 0
  }
`
```